runtime/auth/http-auth-aws/common/test/aws/smithy/kotlin/runtime/http/auth/AwsHttpSignerTestBase.kt

```diff
@@ -41,6 +41,7 @@ public abstract class AwsHttpSignerTestBase(
 ) {
     private val testCredentials = Credentials("AKID", "SECRET", "SESSION")
 
+    @Suppress("DEPRECATION")
     private fun buildOperation(
         requestBody: String = "{\"TableName\": \"foo\"}",
```

Comment:
**Question**: In general, this PR suppresses a lot of now-deprecated patterns in existing tests. Shouldn't we update those tests to use the modern methods?

Reply:
I updated many of them but some of them. use `UnitSerializer` or `IdentitySerializer` and I didn't feel like re-writing them. The changes we made here are an optimization, there isn't anything wrong with the old interfaces other than the way they are used in codegen introduces more suspend paths than is actually necessary.